### PR TITLE
Crm457 1459/adjustments calculations 2

### PR DESCRIPTION
**Pull request by dwhenry**
## Description of change
Add Work item adjustments screen

 [Link to relevant ticket](https://dsdmoj.atlassian.net/browse/CRM457-1459)

Screenshot 2024-05-22 at 16 17 40

screen shot (slightly different from prototype) confirmed with @mary-bowden

### CODE REVIEW

File: app/views/nsm/steps/view_claim/show.html.erb [hunk 1]
```diff
@@ -21,6 +21,11 @@
         <li class="moj-sub-navigation__item">
           <%= link_to t('.tabs.claimed_costs'), nsm_steps_view_claim_path(@claim, section: 'claimed_costs'), class: 'moj-sub-navigation__link', 'aria-current': (@section == :claimed_costs ? "page" : nil) %>
         </li>
+        <% if @claim.status == 'part_grant' %>
```
Comment (jsugarman) [inline app/views/nsm/steps/view_claim/show.html.erb:24]: the statuses have enums you could use but either way

```suggestion
        <% if @claim.part_grant? %>
```

File: spec/system/nsm/view_claim_spec.rb [hunk 2]
```diff
@@ -51,6 +51,20 @@
     end
   end
 
+  # TODO: update this to have adjusted values as well...
```
Comment (jsugarman) [inline spec/system/nsm/view_claim_spec.rb:54]: **nit** Is there a ticket for this todo

**Review by jsugarman — APPROVED** [review]
lots going on in the views but maybe could be refactored another time 👍

File: app/views/nsm/steps/view_claim/show.html.erb (hunk 1 above)
Comment (dwhenry) [reply app/views/nsm/steps/view_claim/show.html.erb:24]: let me look at this as I tried it and it didn't work... :(

**Pull request merged**